src/static/styles/partials/_reset.css

```
@@ -12,7 +29,7 @@
 
 /* Base body styling */
 body {
-    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', system-ui, sans-serif;
+    font-family: Jost, -apple-system, BlinkMacSystemFont, 'Segoe UI', system-ui, sans-serif;
     font-display: swap;
```

Stylelint (17.5.0)

[error] Unexpected unknown property "font-display" (property-no-unknown)

(property-no-unknown)